blog/2016-10-06-fusetooling-release-mars.adoc

```diff
@@ -0,0 +1,45 @@
+= JBoss Fuse Tooling released for Eclipse Mars 
+:page-layout: blog
+:page-author: lhein
+:page-tags: [fuse, camel, integration, jbosstools, devstudio, jbosscentral]
+
+
```

Comment:
In your blog posts you simply put this in:

Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/[Red Hat Developers Program blog]